static-assets/ng-views/admin-repository.html

```diff
@@ -392,13 +392,14 @@ <h3 class="modal-title">{{dialogTitle}}</h3>
 
     <div class="modal-footer">
       <button
+        type="button"
         class="btn btn-default" ng-click="hideModal()"
       >{{ 'common.CANCEL_LABEL' | translate }}
       </button>
       <button
         class="btn btn-primary text-capitalize"
-        ng-click="okModalFunction(repo)"
-        ng-disabled="createNameForm.$invalid"
+        type="submit"
+        ng-class="{'disabled': createNameForm.$invalid}"
```

Comment:
styles were not needed since disabled class matches bootstrap disabled styles (only styles, not behavior)